project_task_stage_change_restriction/tests/test_stage_change_restriction.py

```
@@ -0,0 +1,255 @@
+            vals["stage_id"] = stage.id
+        return self.task_tpl.copy(vals).with_user(acting_user)
+
+    def _ok_move(self, task, user, stage):
+        task.with_user(user).write({"stage_id": stage.id})
+        self.assertEqual(task.stage_id, stage)
+
+    def _fail_move(self, task, user, stage):
+        with self.assertRaises(UserError):
+            task.with_user(user).write({"stage_id": stage.id})
+
+    def _ok_create(self, creator, stage, *, assignees=None):
```

Ruff (0.11.9)

Missing return type annotation for private function `_ok_create`

Add return type annotation: `None`

(ANN202)